content/open-source/guides/learn-godot/beginner/index.md

```diff
@@ -0,0 +1,106 @@
+
+Now is a great time to learn game development.
+
+## Why Godot? What is Godot?
+
+[Godot](https://godotengine.org/) is a 2D and 3D game engine with a fully-fledged editor, but it is different from the other big game engines out there. It is **Free and Open Source Software**, funded by the community, backed by large organizations like Mozilla or Microsoft, and lead by a team of passionate developers shooting for the stars. 
+
+With **over 1000 contributors** at the time of writing, Godot is becoming more and more competitive even compared to the most significant engines in the games industry.
+
+With separate 2D and 3D engines, an innovative design, it is already packed with a large number of features making 2D game creation fun and rewarding. The team is now working hand-in-hand with the [Vulkan](https://www.khronos.org/vulkan/) representatives to provide cutting-edge 3D in version 4.0.
+
+Godot uses the permissive [MIT](https://opensource.org/licenses/MIT) license. This means there's no limit to what an excellent team of developers can do with it, even for commercial projects.
+
+## Can I make my dream game?
+
+Game creation takes time and effort. Game developers are people who continuously learn new tricks and learn to enjoy the creation process.
+
+I cannot tell whether you will be able to create your dream project or not, depending on how crazy it is. Still, you can undoubtedly make games that people will enjoy, even alone. It will just take some time and dedication.
+
+> Great things are not done by impulse, but by a series of small things brought together.
+> 
+> Vincent Van Gogh
+
+
+## Getting started with game development
+
+Many enthusiasts starting on a journey to learn a new topic or skill **quickly become swamped by the sheer amount of information out there**. Nowadays, the problem isn't access to information as it once was, but rather **too much scattered information**. There's also the problem that most of it **isn't exactly respectable quality**, because anyone can put anything on the internet without any filters and quality control.
+
+That said, **we, here at [GDQuest](https://www.gdquest.com/), have done the work for you** by creating this **curated list of free online resources** that get you from **no-programming experience to your first full-blown game projects!**
+
+_Buckle your seatbelts, it going to be an awesome ride!_
+
+### Introduction: the first step in a thousand mile journey
+
+This section is for the total beginner that has little to no idea about the game industry nor the game creation process. It introduces high-level concepts that give some insights into what is to be expected.
+
+1. [GDQuest](https://www.gdquest.com/)'s very own starter guide - [Become a Game Developer: How to Learn to Code]({{< ref "tutorial/game-design/godot/become-a-game-creator/first-steps-programming/index.md" >}})
```

Comment:
You can remove the links to the website for GDQuest.

Reply:
Done.